app/styles/stylesheet.css

```diff
@@ -48,33 +41,86 @@ table {
 Theme Styles
 *******************************************************************************/
 
+html {
+  background: #fff;
+}
+
 body {
   box-sizing: border-box;
-  color:#373737;
-  background: #212121;
+  color: #333;
+  
   font-size: 16px;
-  font-family: 'Myriad Pro', Calibri, Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans Light',X-LocaleSpecific-Light,'Open Sans',X-LocaleSpecific,sans-serif;
   line-height: 1.5;
   -webkit-font-smoothing: antialiased;
 }
 
+main {
+  padding-bottom: 40px;
+}
+
+footer {
+  background: #fff;
+  text-align: center;
+  padding: 20px 0;
+}
+
+.wrapper {
+  background: radial-gradient(ellipse at center,#eaeff2 0,rgba(212,221,228,0.5) 50%),linear-gradient(to bottom,#cae1f4 0,rgba(125,185,232,0) 100%);
+  padding: 0 20px;
+
+  border-top: 2px solid #fff;
+}
+
+#tabzilla {
+  position: relative;
+  float: right;
+}
+
+#tabzilla:before {
+  position: absolute;
+  display: block;
+  left: 28px;
+  top: 0;
+  width: 88px;
+  height: 26px;
+  content: '';
+  background-color: transparent;
+  z-index: 1;
+}
+
+#tabzilla a {
+  background-image: url("https://www.mozilla.org/media/img/tabzilla/tabzilla-static-high-res.c4e717b903f2.png");
```

Comment:
Is this URL stable? I wonder if we should copy the file into our project, given what looks like a hash in the filename, which presumably will change if the file is ever updated.